6_visualize/src/prep_legend_fun.R

```diff
@@ -1,23 +1,43 @@
 
-prep_legend_fun <- function(precip_bins){
+prep_legend_fun <- function(precip_bins, legend_styles){
 
   plot_fun <- function(){
     coord_space <- par()$usr
     bin_w_perc <- 0.05 # percentage of X domain
     bin_h_perc <- 0.02 # *also* percentage of X domain
     bin_w <- bin_w_perc * diff(coord_space[c(1,2)])
     bin_h <- bin_h_perc * diff(coord_space[c(1,2)])
-    xright <- coord_space[2]
+
+    # THIS IS ALL right justified for now...
```

Comment:
yeah, we may need to figure out how to move it around for storms in different parts of the country...